app/src/main/java/woowacourse/movie/moviedetail/MovieDetailActivity.kt

```diff
@@ -0,0 +1,198 @@
+
+        runningTime.text = movie.runningTime.toString()
+        description.text = movie.description
+    }
+
+    private fun setNumberOfPeople() {
+        val booker = findViewById<TextView>(R.id.number_of_people)
+        booker.text = numberOfBooker.toString()
+        clickDecreaseBtn(booker)
+        clickIncreaseBtn(booker)
+    }
+
+    private fun clickDecreaseBtn(booker: TextView) {
+        val minusBtn = findViewById<Button>(R.id.minus_people)
+
+        minusBtn.setOnClickListener {
+            numberOfBooker -= 1
+            if (numberOfBooker <= MIN_BOOKER_NUMBER) {
+                numberOfBooker = MIN_BOOKER_NUMBER
+            }
+            booker.text = numberOfBooker.toString()
+        }
+    }
+
+    private fun clickIncreaseBtn(booker: TextView) {
+        val plusBtn = findViewById<Button>(R.id.plus_people)
+
+        plusBtn.setOnClickListener {
+            numberOfBooker += 1
+            if (numberOfBooker >= MAX_BOOKER_NUMBER) {
+                numberOfBooker = MAX_BOOKER_NUMBER
+            }
+            booker.text = numberOfBooker.toString()
+        }
+    }
+
+    private fun clickBookBtn(movie: Movie) {
+        val bookBtn = findViewById<Button>(R.id.book_button)
+        bookBtn.setOnClickListener {
+            val selectedDate = LocalDate.parse(selectDateSpinner.selectedItem.toString())
+            val selectedTime = LocalTime.parse(selectTimeSpinner.selectedItem.toString())
+            val ticket =
+                Ticket(
+                    TICKET_PRICE,
+                    LocalDateTime.of(selectedDate, selectedTime),
+                    numberOfBooker,
+                )
+            val intent = Intent(this, TicketActivity::class.java)
+            intent.putExtra(TICKET_KEY, ticket)
+            intent.putExtra(MOVIE_KEY, movie)
+            startActivity(intent)
+        }
+    }
+
+    override fun onOptionsItemSelected(item: MenuItem): Boolean {
+        return when (item.itemId) {
+            android.R.id.home -> {
+                finish()
+                true
+            }
+            else -> super.onOptionsItemSelected(item)
+        }
+    }
+
+    private fun setDateSpinner(date: RunningDate) {
+        val spinnerAdapter = SpinnerAdapter(this)
+        selectDateSpinner.adapter = spinnerAdapter.getDateSpinnerAdapter(date)
+        selectDateSpinner.setSelection(dateSpinnerPosition)
+
+        selectDateSpinner.onItemSelectedListener = object : AdapterView.OnItemSelectedListener {
+            override fun onItemSelected(
+                parent: AdapterView<*>?,
+                view: View?,
+                position: Int,
+                id: Long,
+            ) {
+                setTimeSpinner(
+                    spinnerAdapter,
+                    LocalDate.parse(selectDateSpinner.getItemAtPosition(position) as String),
+                )
+            }
+
+            override fun onNothingSelected(parent: AdapterView<*>?) {
+            }
```

Comment:
```suggestion
            override fun onNothingSelected(parent: AdapterView<*>?) = Unit
```
구현부가 없는 함수는 Unit을 활용할 수 있어요